app/models/knight.rb

```diff
@@ -9,7 +9,7 @@ def display
   end
 
   def is_valid_move?(board, stop)
```

Comment:
Missed it the first time but the method should be valid_move? instead of is_valid_move?